app/enterprise/1.3-x/developer-portal/working-with-templates.md

```diff
@@ -302,3 +302,1181 @@ This is my second post!
+- [`each`](#eachlist_or_table) - Commonly used helper to iterate over lists / tables.
+- [`print`](#printany) - Commonly used helper to print lists / tables.
+- [`markdown`](#printany) - Commonly used helper to print lists / tables.
+- [`json_decode`](#json_decode) - Decode JSON to Lua table.
+- [`json_encode`](#json_encode) - Encode Lua table to JSON.
+
+
+## Objects
+
+- [`portal`](#portal) - The portal object refers to the current workspace portal being accessed.
+- [`page`](#page) - The page object refers to the currently active page, and it's contents.
+- [`user`](#user) - The user object represents the currently logged in developer accessing the Kong Portal.
+- [`theme`](#theme) - The theme object represents the currently active theme, and it's variables.
+- [`tbl`](#tbl) = Table helper methods. Examples: `map`, `filter`, `find`, `sort`
+- [`str`](#str) = String helper methods. Examples: `lower`, `upper`, `reverse`, `endswith`
+- [`helpers`](#helpers) - Helper functions simplify common tasks or provide easy shortcuts to Kong Portal methods.
+
+
+## Terminology / Definitions
+
+- `list` - Also referred to commonly as an array (`[1, 2, 3]`) in lua is a table-like object (`{1, 2, 3}`). Lua list index starts at `1` not `0`. Values can be accessed by array notation (`list[1]`).
+- `table` - Also commonly-known-as an object or hashmap (`{1: 2}`) in lua looks like (`{1 = 2}`). Values can be accessed by array or dot notation (`table.one or table["one"]`).
+
+# Globals
+## `l(key, fallback)`
+
+#### Description
+
+> Returns the current translation by key from the currently active page.
+
+#### Return Type
+{% raw %}
+
+```lua
+string
+```
+{% endraw %}
+#### Usage
+
+
+##### `content/en/example.txt`
+
+{% raw %}
+```yaml
+---
+layout: example.html
+
+locale:
+  title: Welcome to {{portal.name}}
+  slogan: The best developer portal ever created.
+---
+```
+{% endraw %}
+
+
+##### `content/es/example.txt`
+
+{% raw %}
+```yaml
+---
+layout: example.html
+
+locale:
+  title: Bienvenido a {{portal.name}}
+  slogan: El mejor portal para desarrolladores jamás creado.
+---
+```
+{% endraw %}
+
+
+##### `layouts/example.html`
+
+{% raw %}
+```lua
+<h1>{* l("title", "Welcome to" .. portal.name) *}</h1>
+<p>{* l("slogan", "My amazing developer portal!") *}</p>
+<p>{* l("powered_by", "Powered by Kong.") *}</p>
+```
+{% endraw %}
+
+
+##### Output when on `en/example`
+
+{% raw %}
+```html
+<h1>Welcome to Kong Portal</h1>
+<p>The best developer portal ever created.</p>
+<p>Powered by Kong.</p>
+```
+{% endraw %}
+
+
+##### Output when on `es/example`
+
+{% raw %}
+```html
+<h1>Bienvenido a Kong Portal</h1>
+<p>El mejor portal para desarrolladores jamás creado.</p>
+<p>Powered by Kong.</p>
+```
+{% endraw %}
+
+
+#### Notes
+
+- `l(...)` is a helper from the `page` object. It can be also accessed via `page.l`. However `page.l` does not support template interpolation (aka `{{portal.name}}` will not work.)
```

Comment:
```suggestion
- `l(...)` is a helper from the `page` object. It can be also accessed via `page.l`. However, `page.l` does not support template interpolation ( `{{portal.name}}` will not work.)
```

aka means also known as and is not appropriate